package.json

```diff
@@ -50,7 +50,7 @@
     "eslint-plugin-import": "^2.22.0",
     "jsdoc": "^3.6.4",
     "mocha": "^7.2.0",
-    "nock": "9.3.2",
+    "nock": "^11.9.1",
```

Comment:
does the change upgrade `nock`, instead of `mocha` as in the commit message? :)

Reply:
indeed, good catch